.github/workflows/_tauri.yml

```diff
@@ -84,7 +84,7 @@ jobs:
       - name: Sign the MSI
         if: ${{ runner.os == 'Windows' }}
         shell: bash
-        run: ../../scripts/build/sign.sh ../target/release/bundle/msi/Firezone_${{ env.VERSION }}_x64_en-US.msi
+        run: ../../scripts/build/sign.sh ../target/release/bundle/msi/firezone-client-gui_${{ env.VERSION }}_x64_en-US.msi
```

Comment:
This _should_ be fine since the rename script still renames it to $BINARY_DEST_PATH